tests/61push/03_unread_count.pl

```diff
@@ -25,6 +25,8 @@
          my ( $event_id ) = @_;
 
          matrix_advance_room_receipt_synced( $user1, $room_id, "m.read" => $event_id );
+      })->then( sub {
+         delay( 0.5 )
```

Comment:
ugh

Reply:
Oh, I had forgotten about that :/

Reply:
In fact, I think I'm being a crank and uploaded something I shouldn't have...